addons/stock_account/migrations/9.0.1.1/noupdate_changes.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='utf-8'?>
+<openerp>
+  <data>
+    <record id="default_cost_method" model="ir.property">
```

Comment:
This is resetting the default values. Are you sure this is correct?

Reply:
Yes...